proposals/dictionary-expressions.md

```diff
@@ -532,6 +532,29 @@ Options include:
 2. Use the target type implementation of `IDictionary<K, V>.this[K] { get; set; }`.
 3. Use the accessible indexer that matches the signature `V this[K] { get; set; }`.
 
+### `dynamic` elements
+
+Related to the previous question, how should the compiler bind to the indexer when the element expression has `dynamic` type, or when either the key or value is `dynamic`?
+
+For reference, with non-dictionary targets, an element with `dynamic` type, the compiler binds to the applicable `Add(value)` method at runtime.
+
+For dictionary targets, the situtation is more complicated because we have key-value pairs to consider. ...
```

Comment:
`...` implies unfinished, but just in case:

```suggestion
For dictionary targets, the situation is more complicated because we have key-value pairs to consider. ...
```

Reply:
Thanks. I've since removed the question about `dynamic` elements and clarified the steps in the Construction section so I think the question no longer applies.